### PR TITLE
All files for Ancestry.com and Wink Case Studies

### PR DESCRIPTION
**Pull request by kbarnard10**
Adding HTML, CSS, images, and update index file for the Ancestry.com case study.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4412)

### CODE REVIEW

**Comment by kbarnard10** [comment]
@steveperry-53 @janetkuo Is it possible to take a look at this today?

**Comment by steveperry-53** [comment]
Yes, I'll take a look today.

File: case-studies/ancestry.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+---
+<!DOCTYPE html>
+<html>
+  <head>
+
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="style_ancestry.css">
+
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: I think this should be href="/css/style_ancestry.css".

**Comment by steveperry-53** [comment]
@kbarnard10, Added one comment about css path. I think you need to add /css/.

File: case-studies/ancestry.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: 90 million of family trees -> 90 million family trees
DNA people -> people

File: case-studies/ancestry.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+</div>
+
+<div class="col2">
+  <h2>Solution</h2>
+
+  The company is transitioning to cloud native infrastructure, using <a href="https://www.docker.com">Docker</a> containerization, <a href="https://kubernetes.io">Kubernetes</a> orchestration and <a href="https://prometheus.io">Prometheus</a> for cluster monitoring.<br>
+<br>
+<h2>Impact</h2>
+  "Every single product, every decision we make at Ancestry, focuses on delighting our customers with intimate, sometimes life-changing discoveries about themselves and their families," says MacKay. "As the company continues to grow, the increased productivity gains from using Kubernetes has helped Ancestry make customer discoveries faster. With the move to Dockerization for example, instead of taking between 20 to 50 minutes to deploy a new piece of code, we can now deploy under a minute for much of our code. We’ve truly experienced a significant time savings in addition to the various features and benefits from cloud native and Kubernetes-type technologies."
+
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: deploy under -> deploy in under

File: case-studies/ancestry.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    The stability of that Shaky Leaf was a signal for MacKay and his team that their decision to embrace cloud native technologies was the right one for the company. With a private data center, Ancestry built its website (which launched in 1996) on hundreds of services and technologies and a traditional deployment methodology. "It worked well for us in the past, but the sum of the legacy systems became quite cumbersome in its processing and was time-consuming," says MacKay. "We were looking for other ways to accelerate, to be more agile in delivering our solutions and our products."
+</div>
+
+<div class="banner3">
+  <div class="banner3text">
+"And when it [Kubernetes] went live smoothly in early 2016, 'our deployment time for this service literally was cut down from 50 minutes to 2 or 5 minutes,' MacKay adds. 'The development team was just thrilled because we're focused on supplying a great experience for our customers. And that means features, it means stability, it means all those things that we need for a first-in-class type operation.'"
+  </div>
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: Repeated paragraph: "And when it went live ..."

File: case-studies/ancestry.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  Figuring out the best deployment operations for Ancestry was a big part of the work it took to adopt cloud native infrastructure. "We want to make sure the process is easy and also controlled in the manner that allows us the highest degree of security that we demand and our customers demand," says MacKay. "With Kubernetes and other products, there are some good solutions, but a little bit of glue is needed to bring it into corporate processes and governances. It's like having a set of gloves that are generic, but when you really do want to grab something you have to make it so it's customized to you. That's what we had to do."<br><br>
+  Their best practices include allowing their developers to deploy into development stage and production, but then controlling the aspects that need governance and auditing, such as secrets. They found that having one name space per service is useful to have that containment for secrets and config maps. And for their needs, having one container per pod makes it easier to manage and to have a smaller unit of deployment.
+<br><br>
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: name space -> namespace
is useful to have that containment for -> is useful for achieving that containment of

File: case-studies/ancestry.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+As much as he and his team at Ancestry have benefited from what he calls "the goodness and the technical abilities of many" in the community, they've also contributed information about best practices, logged bug issues and other typical open source conversation. And they've been active in attending <a href="https://www.meetup.com/Utah-Kubernetes-Meetup/">meetups</a> to help educate and give back to the local tech community in Utah. Says MacKay: "We're trying to give back as far as our experience goes, rather than just code."
+<br><br>When he meets with companies considering adopting cloud native infrastructure, the best advice he has to give from Ancestry's Kubernetes journey is this: "Start small, but with hard problems," he says. And "you need a patron who understands the vision of containerization, to help you tackle the political as well as other technical roadblocks that can occur when change is needed."<br><br>
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: Having trouble with "other typical open source conversation." How about, "contributed information about best practices, logged bugs and issues, and participated in the open source conversation."

File: case-studies/ancestry.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+With the changes that MacKay's team has led over the past year and a half, cloud native will be part of Ancestry's technological genealogy for years to come. MacKay has been such a champion of the technology that he says people have jokingly accused him of having a Kubernetes tattoo.<br><br>
+"I really don't," he says with a laugh. "But I'm passionate. I'm not exclusive to any technology; I use whatever I need that's out there that makes us great. If it's something else, I'll use it. But right now I believe in Kubernetes. I believe in containerization. I think if we can get there and establish ourselves in that world, we will be further along and far better off being agile and all the things we talk about, and it'll go forward."<br><br>
+He pauses. "So, yeah, I guess you can say I'm an evangelist for Kubernetes," he says. "But I'm not getting a tattoo!"
+
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: Repeated text: "if we can get there and establish ourselves ..."

File: case-studies/wink.html [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  </head>
+<body>
+
+
+<div class="banner1">
+  <h1>CASE STUDY: <img src="/images/wink_logo.png" class="logo" width="13%" style="margin-bottom:-4px"><br>
+    <div class="subhead">Cloud-Native Infrastructure Keeps Your Smart Home Connected</div>
+  </h1>
+</div>
+
```
Comment (steveperry-53) [inline case-studies/wink.html]: Do you want the top banner to have Kubernetes and its menu at the top? For example, https://kubernetes.io/case-studies/box/.

File: case-studies/wink.html [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+        <br>
+
+      <h2>Solution</h2>
+        Across-the-board use of a Kubernetes-Docker-CoreOS Container Linux stack.<br><br>
+  </div>
+
+  <div class="col2">
+    <h2>Impact</h2>
+      "Two of the biggest American retailers [Home Depot and Walmart] are carrying and promoting the brand and the hardware,” Wink Head of Engineering Kit Klein says proudly – though he adds that "it really comes with a lot of pressure. It’s not a retail situation where you have a lot of tech enthusiasts. These are everyday people who want something that works and have no tolerance for technical excuses.” And that’s further testament to how much faith Klein has in the infrastructure that the Wink team has built. With 80 percent of Wink’s workload running on a unified stack of Kubernetes-Docker-CoreOS, the company has put itself in a position to continually innovate and improve its products and services. Committing to this technology, says Klein, "makes building on top of the infrastructure relatively&nbsp;easy.”
+  </div>
```
Comment (steveperry-53) [inline case-studies/wink.html]: I don't understand what brand and hardware this refers to. When I read the same text later in the article, then I do understand. But here, out of context, I don't.

File: case-studies/ancestry.html [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+</div>
+
```
Comment (steveperry-53) [inline case-studies/ancestry.html]: Do you want a footer? For example, https://kubernetes.io/case-studies/box/.

Comment (kbarnard10) [inline case-studies/ancestry.html]: Yes - as I understood there was a CSS update the other day that should automatically add it.

File: case-studies/ancestry.html (hunk 4 above)
Comment (kbarnard10) [inline case-studies/ancestry.html]: This should be repeated, one is a banner with the quote called out.

File: case-studies/ancestry.html [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+  The success of Ancestry's first deployment of the hint system on Kubernetes helped create momentum for greater adoption of the technology. "Engineers like to code, they like to do features, they don't like to sit around waiting for things to be deployed and worrying about scaling up and out and down," says MacKay. "After a while the engineers became our champions. At training sessions, the development teams were always the ones saying, 'Kubernetes saved our time tremendously; it's an enabler; it really is incredible.' Over time, we were able to convince our management that this was a transition that the industry is making and that we needed to be a part of it."<br><br>
+  A year later, Ancestry has transitioned a good number of applications to Kubernetes. "We have many different services that make up the rich environment that [the website] has from both the DNA side and the family history side," says MacKay. "We have front-end stacks, back-end stacks and back-end processing type stacks that are in the cluster."<br><br>
```
Comment (kbarnard10) [inline case-studies/ancestry.html]: Should be repeated, one is a banner calling out the quote.

File: case-studies/ancestry.html (hunk 7 above)
Comment (kbarnard10) [inline case-studies/ancestry.html]: This is another called out quote so duplication is ok

File: case-studies/wink.html (hunk 8 above)
Comment (kbarnard10) [inline case-studies/wink.html]: Yes - I was told this should be included in the CSS update pushed last week.

File: case-studies/wink.html (hunk 9 above)
Comment (kbarnard10) [inline case-studies/wink.html]: I think it makes sense in the layout. This is basically just a short outline at the top under the brand logo. But let me know if you still think that's a concern and I'll take another look.

**Comment by kbarnard10** [comment]
@steveperry-53 Took a look at all your comments and made the suggested copy edits.

File: case-studies/wink.html (hunk 9 above)
Comment (steveperry-53) [inline case-studies/wink.html]: Sounds good to me.

**Comment by steveperry-53** [comment]
@kbarnard10, LGTM. Let me know when you're ready for me to merge this.

**Comment by kbarnard10** [comment]
@steveperry-53 Ready when you are!

**Comment by steveperry-53** [comment]
@kbarnard10, Can you update your branch?

File: case-studies/ancestry.html [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+<link rel="stylesheet" type="text/css" href="/css/style_ancestry.css">
+
+<title>Ancestry Case Study</title>
+
+</head>
+
+<body>
+
+<div class="banner1">
+  <h1> CASE STUDY:<img src="http://c.mfcreative.com/i/logo/ancestry-on-dark.svg" class="logo" width="22%" style="margin-bottom:-12px;margin-left:3px;"><br> <div class="subhead">Digging Into the Past With New Technology</div></h1>
```
Comment (tnir) [inline case-studies/ancestry.html]: 🤔 A problem on this external media file link was filed as #10532

**Pull request merged**